assets/javascripts/psique/panels.js.coffee

```diff
@@ -5,3 +5,26 @@ $ ->
 
     $([tab[0], tabpanel[0]]).addClass('active')
     $([tab[0], tabpanel[0]]).siblings().removeClass('active')
+
+$ ->
```

Comment:
I think this JS code should not go here. This file is only for JS element 'panel'. Maybe you would have to create a specific directory only for the styles and JS of the guide itself.